### DISCUSSION TITLE
PID values not updating in autotune but F value is

### DISCUSSION THREAD

**Discussion by zortrop** [iNavFlight/inav#8869] [Q&A] [answered]
Hi I'm new to iNav, so I assume I'm doing something wrong and this is not a bug. 

I have 3 models running iNav 6 RC3 / RC2. I have an OSD layout with the PID/F values. When I run autotune and fly for a while in acro, no other modes enabled, I see the F values updating, but never P,I,D.

In every case, when first configuring iNav I manually increased the I to 10 to make sure it has enough authority to correct on maiden (I saw that suggestion somewhere). Other than that, I didn't change anything from defaults in PID. 

Is there a setting I'm missing? Any tips appreciated!

**Comment by MrD-RC** [comment, marked as answer]
This is working as expected. Only FF and Rates are adjusted by autotune. The base P, I, and D should suit most planes. If they need adjusting, it is more for personal preference.

**Reply by zortrop** [reply]
I see, thanks! 

I'll have to read up to see if/when I might change those values. Looks like there's a healthy discussion of it on the [FWG site](https://inavfixedwinggroup.com/guides/tuning/inav-plane-tuning-masterclass-phase-two-fine-tune-your-plane/). What wasn't clear to me from reading there / elsewhere is that autotune doesn't touch P,I,D.
